mlir/tools/miopen-gen/miopen-gen.cpp

```diff
@@ -1285,34 +1287,32 @@ static void emitMemcpy(OpBuilder &b, mlir::Value src, mlir::Value dst) {
   b.create<CallOp>(loc, memcpyFunc, ValueRange{srcFlat, dstFlat, cstSize});
 }
 
-static void emitPrintTensor(OpBuilder &b, mlir::Value var, bool flag = true) {
-  if (flag) {
```

Comment:
A boolean parameter that controls the execution of a function offends my aesthetic sensibilities.  I removed it entirely and instead controlled whether emitPrintTensor() is called.